.gitattributes

```diff
@@ -1 +1,5 @@
 napari_gui/_version.py export-subst
+*.png filter=lfs diff=lfs merge=lfs -text
+*.jpg filter=lfs diff=lfs merge=lfs -text
+*.gif filter=lfs diff=lfs merge=lfs -text
+docs/images/logo.png !filter=lfs !diff=lfs !merge=lfs -text
```

Comment:
We actually need to include everything inside the [napari/resources/](https://github.com/napari/napari/tree/main/napari/resources) folder. Including the `logo.png` and `loading.gif` copies that are there. The `docs/images/logo.png` that you have there shouldn't need to be excluded.

Hopefully fixing this will fix our tests are ultimately they are erroring with

```pytb
  E       ValueError: Could not find a backend to open `/home/runner/work/napari/napari/napari/resources/logo.png`` with iomode `ri`.
```

Reply:
@sofroniewn is that separation of concerns such that all image files in `napari/resources` SHOULD NOT be under LFS and all image files in `docs` SHOULD

Reply:
yes - to the best of my understanding - all image files in napari/resources are used by the Qt application (i.e. splash screen) and as such need to be included directly in the repo so our CI passes

all image files inside docs are only needed for docs and can live only in LFS

Reply:
That sounds good - I'll update